frontend/device/android/device.lua

```diff
@@ -399,11 +400,15 @@ end
 
 function Device:canExecuteScript(file)
     local file_ext = string.lower(util.getFileNameSuffix(file))
-    if android.prop.flavor ~= "fdroid" and file_ext == "sh"  then
+    if android.prop.flavor ~= "fdroid" and file_ext == "sh" then
         return true
     end
 end
 
+function Device:canApplyPatches()
+    return android.prop.flavor ~= "fdroid" and true or false
```

Comment:
```suggestion
    return android.prop.flavor ~= "fdroid"
```
is enough to return true or false :)